tests/appsec/test_automated_user_and_session_tracking.py

```diff
@@ -73,22 +73,22 @@ def test_user_tracking_auto(self):
             assert meta["_dd.appsec.user.collection_mode"] == "identification"
 
     def setup_user_tracking_sdk_overwrite(self):
+        self.r_login = weblog.post("/login?auth=local", data=login_data(context, USER, PASSWORD))
+
         self.requests = {
-            "before": weblog.post(
-                "/login?auth=local&sdk_trigger=before&sdk_event=success&sdk_user=sdkUser",
-                data=login_data(context, USER, PASSWORD),
+            "before": weblog.get(
+                "/users?user=sdkUser&sdk_trigger=before",
```

Comment:
I feel this is weird having `sdk_trigger `params in this route, as we don't know exactly which auto events it's supposed to happen before or after (whereas with login it's very clear). If it doesn't impact other tracers, I'd remove it 🤔

Reply:
Indeed I think i'll remove it completely for now

Reply:
done!